content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
+   production database:
+
+   #. Scheduled actions are disabled.
+   #. Outgoing mail servers are disabled by archiving the existing ones and adding a fake one.
+   #. Payment providers and delivery carriers are reset to the test environment.
+
+Testing as many of your business flows as possible is strongly recommended to ensure they are
+working correctly and to get more familiar with the new version.
+
+.. admonition:: Basic test checklist
+
+   - Are there views that are deactivated in your test database but active in your production
+     database?
+   - Are your usual views still displayed correctly?
+   - Are your reports (invoice, sales order, etc.) correctly generated?
+   - Are your website pages working correctly?
+   - Are you able to create and modify records? (sales orders, invoices, purchases, users, contacts,
+     companies, etc.)
+   - Are there any issues with your mail templates?
+   - Are your search filters still present?
+   - Can you export your data?
+
+.. spoiler:: Example of end-to-end testing
+
+   - Checking a random product in your product catalog and comparing its test and production data to
+     verify everything is the same (product category, selling price, cost price, vendor, accounts,
+     routes, etc.).
+   - Buying this product (Purchase app).
+   - Confirming the reception of this product (Inventory app).
+   - Checking if the route to receive this product is the same in your production database
+     (Inventory app).
+   - Selling this product (Sales app) to a random customer.
+   - Opening your customer database (Contacts app), selecting a customer (or company), and checking
+     its data.
+   - Shipping this product (Inventory app).
+   - Checking if the route to ship this product is the same as in your production database
+     (Inventory app).
+   - Validating a customer invoice (Invoicing or Accounting app).
+   - Crediting the invoice (issuing a credit note) and checking if it behaves as in your production
+     database.
+   - Checking your reports' results (Accounting app).
+   - Randomly checking your taxes, currencies, bank accounts, and fiscal year (Accounting app).
+   - Making an online order (Website apps) from the product selection in your shop until the
+     checkout process and checking if everything behaves as in your production database.
 
-If you encounter issues or problems in the **production database**, please get in touch with **Odoo
-Support**:
+   This list is **not** exhaustive. Extend the example to your other apps based on your use of Odoo.
 
-#. Connect to our `Odoo Support page <https://www.odoo.com/help>`_.
-#. Under the *Ticket Description* section, select the appropriate type related to your issue but
-   **do not select** the option *An issue related to my upgrade*.
+If you face an issue while testing your upgraded test database, you can request the assistance of
+Odoo via the `support page <https://odoo.com/help>`_. In any case, it is essential to report any
```

Comment:
Personally, I think that if the customer clicks on this link after reading that paragraph, because he encounters an issue with the testing of his upgraded test database, and that he read `an issue while testing your upgraded test database`, he would naturally check the option (currently named as) ` An issue related to my future upgrade (I am testing an upgrade)`

Having a screenshot would add yet another image to maintain throughout the year when name changes, as you can see even in the current documentation, the screenshot does not match exactly the current selection field of odoo.com/help.

IMO, I think there will be little to no user that will read that sentence, click the button, and select another option